homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -51,67 +61,121 @@
             cv.ensure_list, [vol.In(list(SENSORS))]
         ),
         vol.Optional(CONF_NAME, default=""): cv.string,
+        vol.Optional(CONF_SCAN_INTERVAL, default=60): cv.time_period,
```

Comment:
Scan interval is already part of the platform schema you are extending here.

To set a different scan interval on this platform, only the `SCAN_INTERVAL` constant needs to be set.

Reply:
I set the `SCAN_INTERVAL` constant and removed the `CONF_SCAN_INTERVAL` from the `PLATFORM_SCHEMA`.  I'm just using the `SCAN_INTERVAL` constant directly when setting the `update_interval` attribute for the `coordinator`. 

I know [ADR-0010](https://github.com/home-assistant/architecture/pull/366) says that changes to YAML configurations will no longer be excepted. In the future how can I make the `SCAN_INTERVAL` configurable?

Reply:
Well, essentially, we don't want them to be configurable. It should have a sane default.
Nevertheless, if there is a good reason to still make them configurable, if migrated to a configuration flow (ADR-0010), one can add an OptionFlow to allow for setting that.

Reply:
All my changes are made.  Thank you for your help!